public/src/main/java/com/revenuecat/purchases/CustomerInfo.kt

```diff
@@ -49,14 +54,17 @@ data class CustomerInfo constructor(
 
     /**
      * @return Set of active subscription productIds
+     *
+     * For Google subscriptions, productIds will be subscriptionId:basePlanId
+     * For Amazon subscriptions, productIds will be termSku
      */
     @IgnoredOnParcel
     val activeSubscriptions: Set<String> by lazy {
```

Comment:
really wish this was activeSubscriptionIds...probably not worth the rename at this point though